sdiff/parser.py

```diff
@@ -212,7 +212,7 @@ def _remove_ltr_rtl_marks(text):
     return re.sub(r'(\u200e|\u200f)', '', text)
 
 
-def parse(text, parser_cls: Type[MdParser] = MdParser):
+def parse(text, parser_cls: type[MdParser] = MdParser):
```

Comment:
recommended to use `type` over `Type`?

Reply:
this is how pyupgrade --py36-plus works by default: tries to use typing.type which fails

Additionally typing.Type usage seems to be deprecated according to: https://docs.python.org/3.11/library/typing.html#typing

so at this point using type[My_Class] should be way to go